### PR TITLE
Exception handling

### PR DESCRIPTION
**Pull request by nitishgoyal13**

### CODE REVIEW

File: src/main/java/com/flipkart/foxtrot/client/senders/HttpSyncEventSender.java [hunk 1]
```diff
@@ -88,17 +88,10 @@ public void send(final String table, byte[] payload) {
             }
         } catch (FeignException e) {
             logger.error("table={} msg=event_publish_failed", new Object[]{table}, e);
```
Comment (nitishgoyal13) [inline src/main/java/com/flipkart/foxtrot/client/senders/HttpSyncEventSender.java:97]: When we suspended foxtrot yesterday, we were just logging this exception and fabirc topology continued

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 language: java
 jdk:
-  - oraclejdk7
+  - oraclejdk8
```
Comment (santanusinha) [inline .travis.yml]: Move this to openJDK 8

File: src/main/java/com/flipkart/foxtrot/client/senders/QueuedSender.java [hunk 3]
```diff
@@ -112,12 +112,8 @@ public void close() throws Exception {
         private String path;
         private AtomicBoolean running = new AtomicBoolean(false);
 
-        public MessageSenderThread(QueuedSender queuedSender,
-                                   EventSender eventSender,
-                                   BigQueue messageQueue,
-                                   String path,
-                                   EventSerializationHandler serializationHandler,
-                                   int batchSize) throws Exception {
+        MessageSenderThread(QueuedSender queuedSender, EventSender eventSender, BigQueue messageQueue, String path,
```
Comment (santanusinha) [inline src/main/java/com/flipkart/foxtrot/client/senders/QueuedSender.java]: Change back the formatting

**Review by santanusinha — CHANGES_REQUESTED** [review]

File: src/main/java/com/flipkart/foxtrot/client/senders/QueuedSender.java (hunk 3 above)
Comment (nitishgoyal13) [reply src/main/java/com/flipkart/foxtrot/client/senders/QueuedSender.java]: Done

File: .travis.yml (hunk 2 above)
Comment (nitishgoyal13) [reply .travis.yml]: Done

**Pull request closed without merging**